README.md

```diff
@@ -198,6 +202,42 @@ IBIS_SCALAR = {
 }
 ```
 
+# Generating Substrait from Adhoc SQL/Ibis
+The CLI tool for generating substrait plans from adhoc SQL queries and Ibis expression
+is located in the `substrait_consumer/tests/adhoc` directory.  The SQL queries should be 
+written using the same TPCH data used in the integration tests.  This tool will generate 
+the substrait plans for each supported producer and run that plan against all supported consumers.
+
+## How to Use
+Copy your SQL query into `substrait_consumer/tests/adhoc/query.sql`.  
+Copy your Ibis expression into `substrait_consumer/tests/adhoc/ibis_expr.py`
+
+*Note: Do not modify line 1 in ibis_expr.py
```

Comment:
I'm confused what this means.  Is `substrait_consumer/tests/adhoc/ibis_expr.py` a template I copy from?  Or is it the destination I need to save my file as?  If it is a destination then what line 1 is being discussed here?

Reply:
It's the location where the ibis expression should be written.  I'm expecting a specific name for the ibis expression as well as all the TPCH tables to be passed there, that's what the note is supposed to be for.  I can make it more descriptive.